external_search.py

```diff
@@ -52,7 +59,8 @@ def __init__(self, url=None, works_index=None):
             if not url:
                 raise Exception("Cannot connect to Elasticsearch cluster.")
 
-        self.works_index = self.__client.works_index
+        self.works_alias = self.works_index = self.__client.works_index
```

Comment:
The alias is initialized as the index name, so it can always be used for search - even if there isn't actually an alias.